src/main/java/racingcar/Racing.java

```diff
@@ -0,0 +1,40 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Racing {
+    private final int tryTime;
+    private List<RacingCar> racingCars;
+
+    public Racing(int carNum, int tryTime) {
+        racingCars = createRacingCar(carNum);
+        this.tryTime = tryTime;
+    }
+
+    private List<RacingCar> createRacingCar(int carNum) {
+        List<RacingCar> racingCars = new ArrayList<>(carNum);
+        for (int i = 0; i < carNum; i++) {
+            racingCars.add(new RacingCar());
+        }
+        return racingCars;
+    }
```

Comment:
Q. for(int i = 0 ; i<number ; i++) 이런식으로 안쓰고 stream으로 쓰고싶은데 고민되지만 일단 먼저 올려봤습니다. 

A. 아직 시간도 많고 미션도 많이 남았으니 람다식은 천천히 익히시고 적용해주시면 될 것 같아요! 
부담 안가지셔도 됩니다 👋 해당 for문을 람다식으로 변경한다면 다음과 같이 변경할 수 있습니다.
참고하시면 좋을 것 같아요!
```suggestion
    private List<RacingCar> createRacingCar(int carNum) {
       return IntStream.range(0, carNum)
                       .mapToObj(num -> new RacingCar())
                       .collect(Collectors.toList());
    }
```

Reply:
미션도 많이 남아 람다식 천천히 익히라는 말을 해주시니 뭔가 처음부터 완벽하려는것 때문에 힘들었던 부분이 위안이되네요!! 말 그릇과 리액션! 코멘트 덕분에 힘이 납니다. 👍